messages/nl/app.php

```diff
@@ -0,0 +1,200 @@
+    'You haven\'t added a server yet' => 'Je hebt nog geen server toegevoegd',
+    'You seem like a robot' => 'Je lijkt op een robot',
+    'Your servers' => 'Jouw servers',
+    'Your reviews' => 'Jouw recensies',
+    'Your server has been bumped' => 'Je server is gebumpt',
+    'Your server has been deleted' => 'Je server is verwijderd',
+    'Your server has been saved successfully' => 'Je server is succesvol opgeslagen',
+    'Your reply has been deleted' => 'Je reactie is verwijderd',
+    'Your reply has been saved successfully' => 'Je reactie is succesvol opgeslagen',
+    'Your review has been deleted' => 'Je recensie is verwijderd',
+    'Your review has been saved successfully' => 'Je recensie is succesvol opgeslagen',
+    'just now' => 'zojuist',
+    '{0, plural, =1{# day} other{# days}} ago' => '{0, plural, =1{# dag} other{# dagen}}',
+    '{0, plural, =1{# hour} other{# hours}} ago' => '{0, plural, =1{# hour} other{# hours}}',
+    '{0, plural, =1{# minute} other{# minutes}} ago' => '{0, plural, =1{# minuut} other{# minuten}}',
+    '{0, plural, =1{# second} other{# seconds}} ago' => '{0, plural, =1{# seconde} other{# secondes}}',
+    'DISBOARD was made to make everyone enjoy Discord more.<br>We hope you find awesome Discord servers and friends here 😊.' => 'DISBOARD was gemaakt om Discord plezieriger te maken voor iedereen.<br>We hopen dat je hier geweldige Discord servers en vrienden vindt 😊.',
+    /* Welcome Messages on home page */
+    'Discover <span class="discord-logo">Discord</span> Servers' => 'Ontdek <span class="discord-logo">Discord</span> Servers',
+    'We connect <strong>Discord servers</strong> and <b>people</b> together' => 'Wij verbinden <strong>Discord servers</strong> en <b>mensen</b> samen',
+    'Public <span class="discord-logo">Discord</span> Servers' => 'Publieke <span class="discord-logo">Discord</span> Servers',
+    '<strong class="brand">DISBOARD</strong> is the place where you can <strong>list/find Discord servers</strong>.' => '<strong class="brand">DISBOARD</strong> is dè plek waar jij <strong>Discord servers kan plaatsen en zoeken</strong>.',
+    'Find and join some awesome servers listed here.' => 'Zoek en verbind wat gewelige servers vermeld hier.',
+    'Or <a href="{login-url}">login</a> and <strong>add your server if you are a server owner</strong> 😎' => 'Of <a href="{login-url}">meld aan</a> en <strong>voeg je eigen server toe als je de eigenaar bent</strong> 😎',
+    'Add Your Server!' => 'Voeg Je Server Toe!', // The large button
+    /* Categories */
+    'Adventure Games' => 'Avontuur Spellen',
+    'All Games' => 'Alle Spellen',
+    'Anime · Manga' => 'Anime · Manga',
+    'Community' => 'Sociaal',
+    'Computers' => 'Computers',
+    'FPS Games' => 'FPS Spellen',
+    'MMO Games' => 'MMO Spellen',
+    'MOBA Games' => 'MOBA Spellen',
```

Comment:
Optional: add a hyphen (-) instead of a space. It's correct Dutch, even though I think it looks silly.